### PR TITLE
Restplus order

### PR DESCRIPTION
**Pull request by KHafner**
This PR refactors the current service to use Flask-RESTPlus for Shopcart Place-Order Action.

## Checklist:

- [X] I have written test cases for this change, and my code passed the test
- [X] I have attached this PR to an issue/user story

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -475,35 +475,39 @@ def delete_shopcarts(shopcart_id):
...
-    logger.info("Shopcart with ID [%s] is does not exist.", shopcart_id)
-    return not_found("Shopcart Not found")
+# PATH: /shopcarts/{id}}/place-order
+######################################################################
+@api.route('/shopcarts/<int:shopcart_id>/place-order')
+@api.param('shopcart_id', 'The Shopcart identifier')
+class PlaceOrderResource(Resource):
+    """ Place Order action on a Shopcart"""
+    @api.doc('place_order')
+    @api.response(404, 'Shopcart not found or is empty')
```
Comment (iLtc) [inline service/routes.py:485]: Please add a new response for 204

**Review by iLtc — CHANGES_REQUESTED** [review]

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

**Review by gauridhawan — APPROVED** [review]
LGTM!

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201123-203830 "deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201123-203830") to [dev](https://console.bluemix.net/apps/4aa42608-55fd-40d6-b27b-d99aa18a0658?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:dev"), including commit 9435baadecc392cb889db30217cc58dd69fa2e7b

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028 "deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028") to [prod](https://console.bluemix.net/apps/ac06a6b3-040d-43d7-b290-6bfbba3f8c41?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:prod"), including commit 9435baadecc392cb889db30217cc58dd69fa2e7b

**Pull request merged**